src/Perl6/Metamodel/BOOTSTRAP.nqp

```diff
@@ -2854,6 +2850,9 @@ BEGIN {
             sub add_to_cache($entry) {
 #?if jvm
                 return 0 if nqp::capturehasnameds($capture);
+#?endif
+#?if js
```

Comment:
Are you intentionally adding an `#?if js` for some reason, or could the `#?if jvm` just be turned into `#?if !moar`?

Reply:
I should just update the caching on the js backend and get rid of that workaround soon.

Reply:
Merged the ?ifs in the meantime